tests/test_oom_logger.py

```diff
@@ -178,6 +212,20 @@ def docker_inspect():
     }
 
 
+@pytest.fixture
+def containerd_inspect():
+    return {
+        "process": {
+            "env": [
+                "PAASTA_SERVICE=fake_service",
+                "PAASTA_INSTANCE=fake_instance",
+                "PAASTA_RESOURCE_MEM=512",
+                "MESOS_CONTAINER_NAME=mesos-a04c14a6-83ea-4047-a802-92b850b1624e",
```

Comment:
a more representative test would probably be to remove this env var
```suggestion
```
since we no longer actually set this :)